.coveragerc

```diff
@@ -3,9 +3,13 @@
 plugins = Cython.Coverage
 branch = True
 source =
+    arkane
+    Arkane.py
     rmgpy
     rmg.py
-omit = *Test.py
+omit =
+    *Test.py
+    */test_data/*
```

Comment:
Would you like to add the data subfolder in Arkane?

Reply:
Good idea, I'll force push with this change